src/ws/test-remotectlcertificate.c

```diff
@@ -189,6 +190,8 @@ test_valid_selfsigned (TestCase *test,
   g_assert_no_error (error);
   fname = g_dir_read_name (dir);
   g_assert_cmpstr (fname, ==, "0-self-signed.cert");
+  if (sscg_path)
+      g_assert_cmpstr (g_dir_read_name (dir), ==, "0-self-signed-ca.pem");
```

Comment:
"The order of entries returned from this function is not defined, and may vary by file system or other operating-system dependent factors."

Also: what about the case that both openssl and sscg are installed?  Won't we default to using openssl in that case, and then this file won't be here?

Reply:
Good point about the entry order, I'll fix that.

If sscg is installed, it is always used and prefered. So if the -ca file is *not* there, then cert generation with sscg failed, which is exactly what I want to test.

Reply:
Fixed to get along with either entry order.